catalog.html

```diff
@@ -113,10 +113,20 @@ <h2 class="product-sort-title">Сортировка:</h2>
                 <a class="product-sort-link" href="">По функционалу</a>
               </li>
               <li class="product-sort-item">
-                <a class="product-sort-link product-sort-more" href="">Больше</a>
+                <a class="product-sort-link product-sort-more sortActive" href="">
+                  <span class="visually-hidden">Больше</span>
+                  <svg width="12" height="10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
xmlns можно в разметке удалять
```suggestion
                  <svg width="12" height="10" fill="none">
```

Reply:
Исправлено